payroll/views.py

```diff
@@ -103,12 +104,40 @@ def get_context_data(self, **kwargs):
         return context
 
     def median_entity_salary(self):
-        q = Salary.objects.filter(Q(job__position__employer__parent=self.object) | Q(job__position__employer=self.object))
+        q = Salary.objects.filter(
+            Q(job__position__employer__parent=self.object) |
+            Q(job__position__employer=self.object))
```

Comment:
Would you mind dropping this closing parenthesis onto the next line, at the same indentation level as `q = ...`? What you have here is technically correct, but inconsistent with the formatting in the rest of the project.